maintainers/maintainer-list.nix

```diff
@@ -7529,6 +7529,12 @@
     githubId = 127353;
     name = "Geoffrey Huntley";
   };
+  giang = {
```

Comment:
please move these changes to another commit like `maintainers: add giang`

Reply:
Moved to https://github.com/NixOS/nixpkgs/pull/318722

Reply:
Ah…
I mean another commit in this PR. We should not add maintainer in an independent PR.